tests/Fixtures/Views/rawAndEscaping.php

```diff
@@ -2,7 +2,7 @@
 
 use Tempest\View\GenericView;
 
-/** @var GenericView $this */ ?>
+/** @var GenericView|stdClass $this */ ?>
```

Comment:
@brendt - Can you just confirm this one?

Reply:
for clarification it solves the error caused by `<?= $this->property ?>` . `property` is a property of `GenericView`  thus this will error. By unioning `stdClass` we can add / access any property without phpstan complaining.

Reply:
Wouldn't PHPStan know that there's a magic getter on GenericView?

Reply:
It knows, but you will still get the `Access to an undefined property Tempest\View\GenericView::$property.` error because you should than still define which properties one can get (instead of just any). "normally" you whouldn't be able to get just about anything, there would still be a limit. But we do not know this limit in this case.

Reply:
Gotcha, we were ignoring the error before — right? I'm fine with this docblock, seems like a good solution :)